packages/caliper-cli/lib/lib/config.yaml

```diff
@@ -60,10 +60,14 @@ sut:
             packages: ['fabric-client@1.4.6', 'fabric-protos@2.0.0-snapshot.1', 'fabric-network@1.4.6']
         1.4.7:
             packages: ['fabric-client@1.4.7', 'fabric-protos@2.0.0-snapshot.1', 'fabric-network@1.4.7']
-        1.4.8: &fabric-latest
+        1.4.8:
             packages: ['fabric-client@1.4.8', 'fabric-protos@2.0.0-snapshot.1', 'fabric-network@1.4.8']
-        2.1.0: &fabric-latest-v2
+        1.4.11: &fabric-latest
```

Comment:
might as well add the 9/10 patch levels here. I think long term we need to consider if we only support latest patch levels for minor versions (only bind to 1.4.latest, 2.1.latest, etc)